packages/discovery-provider/plugins/pedalboard/apps/trending-challenge-rewards/src/sdk.ts

```diff
@@ -0,0 +1,40 @@
+import { Configuration, DiscoveryNodeSelector, SolanaRelay, sdk } from '@audius/sdk' 
+
+
+const makeDiscoveryNodeSelector = (allowlist?: string[]) => {
+  return new DiscoveryNodeSelector({
+    allowlist: allowlist ? new Set(allowlist) : undefined,
+  })
+}
+
+const solanaRelay = new SolanaRelay(
+  new Configuration({
+    basePath: '/solana',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    middleware: [
+      {
+        pre: async (context) => {
+          const endpoint = 'https://discoveryprovider.audius.co'
```

Comment:
Should this be a config variable?

Reply:
it could be but it is strictly on DN1 atm

Reply:
(it's the only one running solana relay plugin)